dns_check/tests/test_dns_check.py

```diff
@@ -0,0 +1,154 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+from .mocks import MockTime, success_query_mock, nxdomain_query_mock
+# stdlib
+import mock
+
+# 3p
+from dns.resolver import Resolver, Timeout
+
+# project
```

Comment:
Can you please remove the comments around imports everywhere? That's a legacy decision we're getting rid of :slightly_smiling_face: